_posts/2016-07-20-Automation-testing-with-postman.md

```diff
@@ -23,13 +23,13 @@ Select the “Add” button on the modal that is presented to you.  Add `url htt
 ![Alt text](/img/postman-2016/manage_environment_values.jpg)
 <br/>
 <br/>
-Finally select the "Production" environment in the environment dropdown.
+Finally select the "Production" environment in the environment dropdown and let's create our first request.
 <br/>
 <br/>
 ![Alt text](/img/postman-2016/manage_environment_production.png)
 
 ### Creating a POST request
-First let's enter `{% raw %}{{url}}/WayneS/Calculator/0.1.0{% endraw %}` in the request field and change the method from GET to POST.  We also need some headers so add `Content-Type application/json`, `Authorization {% raw %}Simple {{key}}{% endraw %}` aswell.  Here we are using the environment variable `{% raw %}{{url}}{% endraw %}` and `{% raw %}{{key}}{% endraw %}` so that when we switch environments the variables are replaced accordingly.  The `{% raw %}{{...}}{% endraw %}` format can only be used in the request URL/URL params/Header values/form-data/url-encoded values/Raw body content/Helper fields.
+Enter `{% raw %}{{url}}/WayneS/Calculator/0.1.0{% endraw %}` in the request field and change the method from GET to POST.  We need to add some additional headers aswell so add `Content-Type application/json`, `Authorization {% raw %}Simple {{key}}{% endraw %}`.  Ass youo can see we are using the environment variable `{% raw %}{{url}}{% endraw %}` and `{% raw %}{{key}}{% endraw %}` so that when switching environments these variables will be replaced.  The `{% raw %}{{...}}{% endraw %}` format can only be used in the request URL/URL params/Header values/form-data/url-encoded values/Raw body content/Helper fields.
```

Comment:
'Ass youo' can see we are using the environment variable...
As you can see, we are using the environment variableS
'so that'... Maybe it's better to rephrase. 'so when/while switching environments, those variables will get replaced.  
Can you maybe replace  "URL/URL params/Header values/form-data/url-encoded values/Raw body content/Helper" with a list?